### ISSUE TITLE
Drape imagery over 3D Tiles 

### ISSUE THREAD

**Issue by OmarShehata** [AnalyticalGraphicsInc/cesium#7591]
This seems to be a popular request this week:

* [3d tiles: texturing by ImageryLayers supported?](https://groups.google.com/d/msg/cesium-dev/8XY8Tbjn0s8/DuC1KPeRAwAJ)
* [3D tiles classification with tiled imagery layers](https://groups.google.com/d/msg/cesium-dev/4RW7P2Akjhs/Lk5Q_xY7GgAJ)

The request is for having untextured 3D Tiles, and using imagery to dynamically texture them. 

I wonder if it would be better to instead have a way for ion to drape and bake that texture in? I imagine this is how it would work if/when terrain is folded into imagery.

**Comment by pjcozzi** [comment]
It will be important to be able to do this at runtime when the 3D Tiles rendering engine is used for rendering terrain; hopefully we can interop the existing imagery layers with a quantized-mesh 3D Tiles tile.

**Comment by Frank-Chan** [comment]
Any new progress?

**Comment by OmarShehata** [comment]
@Frank-Chan it's still on the roadmap but I don't have a timeline for it. In the mean time it is possible to drape textured materials on 3D Tiles, see the "Textured Material" dropdown in this example:

https://sandcastle.cesium.com/index.html?src=Classification%20Types.html

**Comment by OmarShehata** [comment]
Requested over email in the context of draping satellite imagery over Cesium OSM Buildings.

**Comment by hpinkos** [comment]
I don't think this is an ideal solution for texturing buildings.  It's difficult to figure out what the texture should be for the vertical faces.  We want to avoid something that looks like this:

  
![image](https://user-images.githubusercontent.com/3451886/84704553-0a9f8300-af28-11ea-9741-7d175eec3831.png)

**Comment by bkuster** [comment]
Allthough I agree with @hpinkos, that this makes little sense for _photorealistic textures_. We had more then one use case, where we would use some form of WMS/WMTS layer to dynamicall show things like _sensor coverage data_. For these use cases (especially if the data is provided by the minute) this would be a cool feature.

**Comment by delbern310** [comment]
any new progress?

**Comment by qilimazhualuo** [comment]
> I don't think this is an ideal solution for texturing buildings. It's difficult to figure out what the texture should be for the vertical faces. We want to avoid something that looks like this:
> 
> ![image](https://user-images.githubusercontent.com/3451886/84704553-0a9f8300-af28-11ea-9741-7d175eec3831.png)



> use some form of WMS/WMTS layer to dynamicall show things like _sensor coverage data_.

hellow, I want to ask how to put WMS on 3dtiles like this, i'd like to put some pipe image (WMS) to 3dtiles

**Comment by ggetz** [comment]
A community member is looking to do this with Google's Photorealistic 3D Tiles: https://community.cesium.com/t/google-photorealistic-3d-tiles/24587/13?u=gabby_getz

**Comment by ggetz** [comment]
Also requested in https://community.cesium.com/t/draping-wms-over-3dtileset/21631/5.

**Comment by msaeedi50** [comment]
hello every body 
I want in secium draw points.geojason and use glb file for all of points in geojson .
how can use glb for all of coordinate of points in geojson 
please help me

**Comment by ggetz** [comment]
Hi @msaeedi50,

CesiumJS is a big project, so we use GitHub for feature requests and bug tracking exclusively. In the future, please take any questions to the [Cesium Forum](https://community.cesium.com/) where there are members of the community and developers from the team who can help. Thanks!

**Comment by jjhembd** [comment]
Another community member requested this:
https://community.cesium.com/t/how-should-cesium-attach-the-map-layer-to-the-3dtiles/34875/3

**Comment by jjhembd** [comment]
Requested again at https://community.cesium.com/t/xyz-tile-layer-2d-over-photorealistic-3d-tiles-in-recent-version-like-1-123/36456/1

**Comment by alexpaulson1** [comment]
Hi Team - this is a key need for many of our users that has come up recently

Need: Ability to drape rasterized imagery onto 3D tiles (similar to the quantized mesh based terrain)

Thanks!

**Comment by mborjesson-bentley** [comment]
This is also a requirement for iTwin Experience. Our users can add their own 3D Tiles-datasets and WMS-layers, and they will expect the WMS to be draped on top of the 3D Tiles-datasets.

Thanks!

**Comment by ggetz** [comment]
[How Raster Overlays Work in Cesium Native](https://cesium.com/learn/cesium-native/ref-doc/how-raster-overlays-work.html)

**Comment by ggetz** [comment]
For anyone watching this issue, there's a draft PT and some discussion around approach happening in https://github.com/CesiumGS/cesium/pull/12567.

**Comment by ggetz** [comment]
Initial implementation has been added in https://github.com/CesiumGS/cesium/pull/12567! @javagl will follow up by opening new issues for known next steps.
